core/src/main/java/fr/sncf/osrd/api/stdcm/STDCMRequest.java

```diff
@@ -131,7 +130,8 @@ public STDCMRequest() {
                 Double.NaN,
                 null,
                 0,
-                0
+                0,
+                43200
```

Comment:
```suggestion
                12 * 3600
```

I'd find that easier to read. But it's very minor, feel free to ignore this